subprojects/common/result/src/main/kotlin/com/avito/android/Result.kt

```diff
@@ -91,6 +93,21 @@ sealed class Result<T> {
         }
     }
 
+    inline fun <R> combine(other: Result<T>, func: (T, T) -> R): Result<R> {
```

Comment:
```suggestion
    inline fun <R> combine(other: Result<T>, onSuccess: (T, T) -> R): Result<R> {
```
?

Reply:
don't think so;
it looks like a function that does actual combining;
`onSuccess` on other hand mimics side effect function name

Reply:
Ok